### PR TITLE
DEV-46435 Grafana 10 error 500 on get api datasources uid uid resources

### PR DESCRIPTION
**Pull request by copyhold**
Skip the datasource elastic version request

### CODE REVIEW

**Comment by copyhold** [comment]
run-tests

**Comment by copyhold** [comment]
run-test

**Comment by copyhold** [comment]
run-test

File: Dockerfile [hunk 1]
```diff
@@ -189,5 +189,9 @@ ARG RUN_SH=grafana/packaging/docker/run.sh
 
 COPY ${RUN_SH} /run.sh
 
+# LOGZ.IO GRAFANA CHANGE :: Add unique file to avoid ECR tag limit
```
Comment (yasmin-tr) [inline Dockerfile:192]: there's no ticket reference to this change?
where is this change coming from? 🤔

**Pull request merged**